cluster.yml

```diff
@@ -1,4 +1,9 @@
 ---
+- hosts: localhost
+  gather_facts: False
+  roles:
+    - bastion-ssh-config
```

Comment:
please add the `localhost` and `bastion` tags for that and document the latter in the docs/ansible.md